includes/core/classes/class-setup.php

```diff
@@ -135,6 +136,31 @@ public function block_category( $block_categories ) {
 		return $block_categories;
 	}
 
+	/**
+	 * Display custom admin notice function
+	 *
+	 * @return void
+	 */
+	public function timezone_check_admin_notice() {
+		$timezone = get_option( 'timezone_string' );
+		if ( $timezone ) {
+			return;
+		}
+		$class   = 'notice notice-error is-dismissible';
+		$string1 = __( 'Please set', 'gatherpress' );
+		$string2 = __( 'your timezone', 'gatherpress' );
+		$string3 = __( 'in order to ensure proper GatherPress settings!', 'gatherpress' );
```

Comment:
I think instead of `GatherPress` should say something like `in order to ensure proper event date and times` or something like that. People may forget the plugin is GatherPress, but know the feature is events.